src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
```

Comment:
I agree that uint64 nanos look a bit cleaner than doubles, but does it really buy us much to change at this point?